crates/rustc_codegen_spirv/src/codegen_cx/entry.rs

```diff
@@ -82,7 +103,7 @@ impl<'tcx> CodegenCx<'tcx> {
         &self,
         span: Span,
         entry_func: SpirvValue,
-        entry_fn_abi: &FnAbi<'tcx, Ty<'tcx>>,
+        arg_abis: &[ArgAbi<'tcx, Ty<'tcx>>],
```

Comment:
I'm not sure this is a good idea, especially if we want to do something with the return type at some point.